stock_shipment_management/model/shipment_plan.py

```diff
@@ -89,8 +89,7 @@ class ShipmentPlan(models.Model):
     )
     from_address_id = fields.Many2one(
         'res.partner', 'From Address',
-        readonly=True,
-        states={'draft': [('readonly', False)]},
+        readonly=True, # You can't update the from address
```

Comment:
./stock_shipment_management/model/shipment_plan.py:92:23: E261 at least two spaces before inline comment 

-> travis is :red_circle: